### DISCUSSION TITLE
Is there a release history?

### DISCUSSION THREAD

**Discussion by tom-doerr** [GothenburgBitFactory/timewarrior#445] [Q&A] [answered]
Is there a change history anywhere? I couldn't find it on Github or in the docs. Would be curious what the new features are.

**Comment by sruffell** [comment, marked as answer]
The only one I'm aware of is the [ChangeLog](https://github.com/GothenburgBitFactory/timewarrior/blob/dev/ChangeLog).

**Reply by tom-doerr** [reply]
Thanks, that's what I was looking for.
